WORKSPACE

```diff
@@ -20,15 +29,25 @@ http_archive(
 )
 
 load("@bazel_gazelle//:deps.bzl", "gazelle_dependencies")
+load("//:go_deps.bzl", "go_deps")
+go_deps()
+
 load("@io_bazel_rules_go//go:deps.bzl", "go_register_toolchains", "go_rules_dependencies")
 
 go_rules_dependencies()
 
-go_register_toolchains(version = "1.22.5")
+go_register_toolchains(
+    nogo = "@//:nogo", 
+    version = "1.22.5",
+)
 
 gazelle_dependencies()
 
 load("//:deps.bzl", "go_dependencies")
 
 # gazelle:repository_macro deps.bzl%go_dependencies
 go_dependencies()
+
+load("@com_github_sluongng_nogo_analyzer//staticcheck:deps.bzl", "staticcheck")
+
+staticcheck()
```

Comment:
This should be called before the first usage of nogo. Move it to line 38.

Reply:
Done, thank you! Couldn't figure out why it works here https://github.com/sluongng/nogo-analyzer/blob/master/examples/staticcheck/WORKSPACE#L54 :(

Reply:
Looks like I was wrong a bit, `staticcheck()`should be called before `gazelle_dependencies()`

```diff
diff --git a/WORKSPACE b/WORKSPACE
index 59c7655..0ee63a0 100644
--- a/WORKSPACE
+++ b/WORKSPACE
@@ -36,16 +36,16 @@ load("@io_bazel_rules_go//go:deps.bzl", "go_register_toolchains", "go_rules_depe
 
 go_rules_dependencies()
 
-# This should be called before the first usage of nogo;
-# @co_honnef_go_tool is defined by `staticcheck()` macro
-load("@com_github_sluongng_nogo_analyzer//staticcheck:deps.bzl", "staticcheck")
-staticcheck()
-
 go_register_toolchains(
     nogo = "@//:nogo",
     version = "1.22.5",
 )
 
+# This should be called before the first usage of nogo;
+# @co_honnef_go_tool is defined by `staticcheck()` macro
+load("@com_github_sluongng_nogo_analyzer//staticcheck:deps.bzl", "staticcheck")
+staticcheck()
+
 gazelle_dependencies()
 
 load("//:deps.bzl", "go_dependencies")

```